index.js

```diff
@@ -120,7 +120,7 @@ function enterWord(word) {
   }
 }
 
-function sizeText(newWerd) {
+function getTextPosition(newWerd) {
   //Calculate proper x coordinate based on text length
   var position = 430;
```

Comment:
I like xCoordinate here better

Reply:
xCoordinate where? instead of getTextXPosition?